src/coreclr/tools/Common/JitInterface/CorInfoInstructionSet.cs

```diff
@@ -752,6 +752,31 @@ private static InstructionSetFlags ExpandInstructionSetByReverseImplicationHelpe
             return resultflags;
         }
 
+        private static Dictionary<(string, TargetArchitecture), string> AllInstructionSetGroups { get; } = new()
+            {
+                { ("x86-x64",    TargetArchitecture.X64),   "sse2" },
+                { ("x86-x64",    TargetArchitecture.X86),   "sse2" },
+                { ("x86-x64-v2", TargetArchitecture.X64),   "sse4.2 popcnt" },
+                { ("x86-x64-v2", TargetArchitecture.X86),   "sse4.2 popcnt" },
+                { ("x86-x64-v3", TargetArchitecture.X64),   "x86-x64-v2 avx2 bmi bmi2 lznct movbe fma" },
+                { ("x86-x64-v3", TargetArchitecture.X86),   "x86-x64-v2 avx2 bmi bmi2 lznct movbe fma" },
+                { ("skylake",    TargetArchitecture.X64),   "x86-x64-v3 pclmul aes" },
+                { ("skylake",    TargetArchitecture.X86),   "x86-x64-v3 pclmul aes" },
```

Comment:
There is a concern that this doesn't match what LLVM exposes here.

Even though it is available on all Skylake processors to my knowledge, it's possibly worth double checking with our friends at Intel/AMD on if its actually guaranteed or not.